### ISSUE TITLE
celltable resolution check

### ISSUE THREAD

**Issue by Manangka** [Deltares/imod-python#85]
In GitLab by @mulderniels on Aug 20, 2019, 14:57

add check in celltable: all dx and dy should be devidable by the resolution

**Comment by Manangka** [comment]
In GitLab by @Huite on Aug 20, 2019, 15:08

The only trick to handle equidistant (float) and non-equidistant (array) dx and dy properly. 
`np.atleast_1d` will do the trick:

```python
if not (np.atleast_1d(dx) % resolution == 0.0).all():
    raise ValueError("...")
```

In terms of testing, you can check if it succesfully raises the error with `pytest.raises`
http://doc.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions

**Comment by Manangka** [comment]
In GitLab by @Huite on Sep 2, 2019, 22:04

closed via commit bc2be22e207d7aa7146b75c493881338b5366b43
